php/register_account_details/register_account_details.php

```diff
@@ -0,0 +1,60 @@
+<?php
+include "../database.php";
+
+$conn = connect_local();
+
+header("Content-Type: application/json");
+
+$request = file_get_contents("php://input");
+
+if(isset($request) && !empty($request))
+{
+    $reqObj = json_decode($request);
+    //Checking for duplicates depending on input
+    switch($reqObj->method)
+    {
+        case "check_duplicates_email":
```

Comment:
I would have several methods for the both duplicate checks and the registration.